lib/ransack/context.rb

```diff
@@ -135,7 +135,7 @@ def ransackable_association?(str, klass)
     end
 
     def ransackable_scope?(str, klass)
-      klass.ransackable_scopes(auth_object).any? { |s| s.to_s == str }
+      klass.ransackable_scopes(auth_object).any? { |s| s.to_sym == str.to_sym }
```

Comment:
Yes, Symbol GC was added in ruby 2.2. This may be OK now, depending on what Ransack wants to support.